pages/docs/tutorials/kafka/bindings-with-kafka.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Adding Kafka information using bindings.
+description: In this tutorial, you'll learn how bindings with Kafka work and different ways to utilize bindings in your AsyncAPI applications.
+weight: 150
+---
+
+## Introduction
+
+In this tutorial, you will learn how bindings with Kafka works. You will learn how to extend your AsyncAPI document with Protocol-specific information, such as how to specify Kafka topic-specific configurations or how to define Kafka message key content.
+
+## Background Context
+Bindings are relevant when integrating Kafka with other programs or services. A binding is a specific agreement or configuration that governs how an application connects to and interacts with Kafka. It can include information such as:
+
+- Which topics should an application read from or write to?
+- How should messages be formatted?
+- What are the rules for interacting with various data or messages?
```

Comment:
Let's turn these items/bullet point list into sentence format and include in the previous paragraph from line 12.